src/core/include/openvino/op/search_sorted.hpp

```diff
@@ -22,7 +22,15 @@ class OPENVINO_API SearchSorted : public Op {
     /// \param values          Values to search indexs for.
     /// \param right_mode      If False, return the first suitable index that is found for given value. If True, return
     /// the last such index.
-    SearchSorted(const Output<Node>& sorted_sequence, const Output<Node>& values, bool right_mode = false);
+    /// \param output_element_type The element type of the output tensor. This is purely an implementation flag, which
+    /// is needed to enable some transformation in ConvertPrecision for CPU plugin(and potentially other plugins as
+    /// well). Setting this flag to element::i32 will result in the output tensor of i32 element type. Setting this flag
+    /// to element::i64 will generally not give any effect, since it will be converted to i32 anyway, at least for CPU
+    /// plugin.
+    SearchSorted(const Output<Node>& sorted_sequence,
```

Comment:
please update a specification with `output_element_type` attribute

Reply:
Specification update can be done in separate PR after CF. AT the same time, this is a pure hack IMO and as such should not be a part of the specification. But please let's skip this discussion after code freeze. At this moment, this change is needed ONLY because some transformation does not work with it.